students/AndrewMiotke/lesson02/assignment/src/charges_calc.py

```diff
@@ -68,15 +36,15 @@ def calculate_additional_fields(data):
         try:
             rental_start = datetime.datetime.strptime(value["rental_start"], "%m/%d/%y")
         except ValueError:
-            WARNING_LOGGER.warning("check rental_end")
+            # add logging
+            logging.error(ValueError)
             rental_end = datetime.datetime.strptime(value["rental_end"], "%m/%d/%y")
             value["total_days"] = (rental_end - rental_start).days
```

Comment:
This should ideally be an absolute value since of the cases will return negative days